tests/installation/opensuse_welcome.pm

```diff
@@ -30,7 +30,7 @@ sub run {
     }
 
     for my $retry (1 .. 5) {
-        wait_screen_change { send_key 'alt-f4' };
+        wait_screen_change { send_key 'alt-f4', timeout => 10 };
```

Comment:
Hey. I've checked the logs and there is warning:
```
[2019-10-28T18:31:39.598 CET] [debug] <<< testapi::send_key(key='alt-f4', do_wait='timeout')
DEPRECATED: wait_idle called, update your test code at /usr/lib/os-autoinst/testapi.pm line 1995.
	testapi::wait_idle() called at /usr/lib/os-autoinst/testapi.pm line 1286
	testapi::send_key("alt-f4", "timeout", 10) called at /var/lib/openqa/pool/12/os-autoinst-distri-opensuse/tests/installation/opensuse_welcome.pm line 33
```

The thing is that `wait_screen_change` doesn't help much to make tests reliable. What about increasing timeout in `check_screen` call below, so we have better chance to catch that window was closed between attempts?
With this change, it's same as having sleep of 10 seconds...

Reply:
It does help, because you avoid to timeout if the `alt-f4` was not missed.

Your 2nd solution will not help, because `check_screen` returns quickly because it matches `opensuse_welcome`, so increasing a timeout there will have no effect.

Reply:
Ah, I've confused the needle, so maybe we should assert that we don't have `opensuse-welcome` instead?
I guess `generic-desktop` will do the trick to assure that pop-up is closed.
In case you want just simple hack, let's remove `wait_screen_change` completely and leave `send_key` with 10 seconds of idling.

Reply:
> Ah, I've confused the needle, so maybe we should assert that we don't have `opensuse-welcome` instead?
> I guess `generic-desktop` will do the trick to assure that pop-up is closed.

That could be a proper solution, but it would be more intrusive.
But I can try this solution.

> In case you want just simple hack, let's remove `wait_screen_change` completely and leave `send_key` with 10 seconds of idling.

In this case, you will loose 10 seconds, even on faster workers, which is not great.